### PR TITLE
chore: Update endo packages

### PR DESCRIPTION
**Pull request by kriskowal**

### CODE REVIEW

File: packages/lockdown/commit-debug.js [hunk 1]
```diff
@@ -64,4 +64,6 @@ lockdown({
   // this may be a development accident that should be fixed before merging.
   //
   // consoleTaming: 'unsafe',
+
+  domainTaming: 'unsafe',
```
Comment (michaelfig) [inline packages/lockdown/commit-debug.js:78]: Add a comment here, please.

File: packages/lockdown/pre.js [hunk 2]
```diff
@@ -138,6 +144,8 @@ export const lockdown = defaultOptions => {
       // this may be a development accident that MUST be fixed before merging.
       //
       // consoleTaming: 'unsafe',
+
+      domainTaming: 'unsafe',
```
Comment (michaelfig) [inline packages/lockdown/pre.js:160]: Same here and above.

File: yarn.lock [hunk 3]
```diff
@@ -17915,6 +17915,11 @@ ses@^0.14.3:
   resolved "https://registry.yarnpkg.com/ses/-/ses-0.14.3.tgz#fadc5a3baa0878adfcbdc8fa383218a821974270"
```
Comment (michaelfig) [inline yarn.lock]: Why wasn't `ses-0.14.3` removed from the `yarn.lock`?  Does something depend on it explicitly?

**Review by michaelfig — COMMENTED** [review]
Getting really close!  I only have minor comments.

If you're ready to merge, let me know and I'll rereview with an eye to approving.

File: yarn.lock (hunk 3 above)
Comment (michaelfig) [reply yarn.lock]: > Why wasn't `ses-0.14.3` removed from the `yarn.lock`? Does something depend on it explicitly?

Ah, looks like `packages/governance/package.json` references `ses` when it shouldn't.

Comment (kriskowal) [reply yarn.lock]: I’ve synced up governance. This is probably something that slipped in with a rebase.

File: packages/lockdown/pre.js (hunk 2 above)
Comment (kriskowal) [reply packages/lockdown/pre.js:160]: Comments added.

**Review by michaelfig — APPROVED** [review]
LGTM!

**Pull request merged**